### ISSUE TITLE
Filename extension selection on Win32

### ISSUE THREAD

**Issue by jcchu** [modula3/cm3#20]
The current Win32 implementation of the “Pathname” interface separates basename and filename extension by the first (instead of the last) dot character.  (For example, `Pathname.LastExt("C:\Windows\system32\ntoskrnl.exe.old")` returns `"exe.old"` instead of `"old"`.)  I couldn’t find any authoritative documentation, but this is in contrast to [the behavior of the .NET framework](https://encrypted.google.com/search?hl=en&gl=us&noj=1&safe=off&gbv=1&strip=1&q=site%3Amsdn.microsoft.com+filename+extension+%28%22last+dot%22+OR+%22last+period%22%29) and the Windows shell.

**Comment by jaykrell** [comment]
You are right it should be last dot, unless before slash, in which case no extension. I thought I cleaned this all up years ago. I’ll look..
